conformance_tests/sysman/test_sysman_device/src/test_sysman_device.cpp

```diff
@@ -127,6 +127,49 @@ TEST_F(
   run_device_hierarchy_child_process();
 }
 
+TEST_F(
+    SYSMAN_DEVICE_TEST,
+    GivenValidDeviceWhenRetrievingSubDevicePropertiesThenValidPropertiesAreReturned) {
+  for (auto device : devices) {
+    auto device_properties = lzt::get_sysman_device_properties(device);
+    uint32_t sub_devices_count = device_properties.numSubdevices;
+
+    if (sub_devices_count > 0) {
+      uint32_t num_subdevices = 0;
+      auto sub_device_properties =
+          lzt::get_sysman_subdevice_properties(device, num_subdevices);
+      EXPECT_EQ(sub_devices_count, num_subdevices);
+      for (uint32_t sub_device_index = 0; sub_device_index < num_subdevices;
+           sub_device_index++) {
+        EXPECT_LT(sub_device_properties[sub_device_index].subdeviceId,
+                  num_subdevices);
+        EXPECT_GT(sub_device_properties[sub_device_index].uuid.id[0], 0);
+      }
+    }
+  }
+}
+
+TEST_F(
+    SYSMAN_DEVICE_TEST,
+    GivenValidDriverWhenRetrievingDeviceHandleFromUUIDThenReturnedDeviceHandleShouldMatchCurrentDeviceHandle) {
+  zes_driver_handle_t driver = lzt::get_default_zes_driver();
+  for (auto device : devices) {
+    auto properties = lzt::get_sysman_device_properties(device);
+    zes_uuid_t uuid = {};
+    for (uint32_t i = 0; i < sizeof(properties.core.uuid.id); i++) {
+      uuid.id[i] = properties.core.uuid.id[i];
```

Comment:
Should "core" field be used to extract uuid in sysman? 
According to the https://spec.oneapi.io/level-zero/latest/sysman/api.html#_CPPv4N23zes_device_properties_t4coreE, it is deprecated.  
@AshwinKumarKulkarni can you please confirm?

Reply:
Other way of getting UUID through extn struct is not supported yet @anvesh-intel